sympy/core/numbers.py

```diff
@@ -211,6 +211,7 @@ def igcd(*args):
 except ImportError:
     def igcd2(a, b):
         """Compute gcd of two Python integers a and b."""
+        a, b = as_int(a), as_int(b)
```

Comment:
I did intentionally not add this line to make `igcd2` work with Python int's only compatibly with `math.gcd`.